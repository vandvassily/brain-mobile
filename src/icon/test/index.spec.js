import { mount } from '@vue/test-utils';
import Icon from '..';

test('should render title slot and match snapshot', () => {
  const wrapper = mount(Icon, {
    propsData: {
      iconName: 'arrow-left',
    },
  });

  expect(wrapper).toMatchSnapshot();
});
