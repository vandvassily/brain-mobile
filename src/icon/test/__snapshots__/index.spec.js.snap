// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render title slot and match snapshot 1`] = `<i class="van-icon van-icon-arrow-left"></i>`;
